### DISCUSSION TITLE
关于mac端，脚本需要重新编译吗

### DISCUSSION THREAD

**Discussion by Tuinnnnn** [hooke007/mpv_PlayKit#151] [mpv 通用]
现在播放视频会报错
`[vapoursynth] Script evaluation failed:
[vapoursynth] Python exception: No attribute with the name svp1 exists. Did you mistype a plugin namespace?
[vapoursynth] 
[vapoursynth] Traceback (most recent call last):
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 2890, in vapoursynth._vpy_evaluate
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 2891, in vapoursynth._vpy_evaluate
[vapoursynth]   File "/Users/tuin/.config/mpv/vs/svpflow-2x.vpy", line 27, in <module>
[vapoursynth]     svps = core.svp1.Super(clip8, super_params)
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 2438, in vapoursynth._CoreProxy.__getattr__
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 2280, in vapoursynth.Core.__getattr__
[vapoursynth] AttributeError: No attribute with the name svp1 exists. Did you mistype a plugin namespace?
[vapoursynth] 
[vapoursynth] could not init VS
Disabling filter vapoursynth.00 because it has failed.
`

**Comment by hooke007** [comment]
发错版块了吧，CM哪来的mac端

`No attribute with the name svp1 exists. Did you mistype a plugin namespace`

没有找到对应名称的插件库。跟编译脚本没有关系，没安装vs的插件而已。
svp的mac版旧插件不好找

**Reply by Tuinnnnn** [reply]
感谢回复，这个插件库我该怎么弄，还没太弄明白。

**Reply by hooke007** [reply]
参考官方文档或者到vs那查相关issue吧，mac端估计没几个人用vs的
http://www.vapoursynth.com/doc/installation.html#linux
